wagtail/models/i18n.py

```diff
@@ -177,7 +177,11 @@ def is_active(self) -> bool:
 class TranslatableMixin(models.Model):
     translation_key = models.UUIDField(default=uuid.uuid4, editable=False)
     locale = models.ForeignKey(
-        Locale, on_delete=models.PROTECT, related_name="+", editable=False
+        Locale,
+        on_delete=models.PROTECT,
+        related_name="+",
+        editable=False,
+        verbose_name=translation.gettext("Locale"),
```

Comment:
I think this should be using `gettext_lazy` rather than `gettext` - see for example the `verbose_name` definitions in `wagtail/models/__init__.py`, which has `from django.utils.translation import gettext_lazy as _`. Otherwise, it will end up doing the translation lookup at load time, and individual users' language preferences won't be respected.

Also, `"locale"` should be all lower-case here, since it sometimes gets incorporated into phrases like "add another locale".

Reply:
Thanks for your quick reply! Good points indeed... Just pushed a fix.